concepts/framework/architecture/administration-concept.md

```diff
@@ -128,5 +128,5 @@ You will find more information about customizing components of the administratio
 
 The Access Control List or ACL in Shopware ensures that by default data can only be created, read, updated or deleted \( CRUD\), once the user has specific privileges for a module. Additionally, one can set up custom roles in the Administrations UI or develop individual privileges with plugins. These roles do have finely granular rights, which every shop operator can set up himself and can be assigned to users. By default, a module of the Administration has a directory called `acl` included. In this directory one will find specific mappings of privileges \(permissions for roles; additional permissions\) for the default roles: `viewer`, `editor`, `creator`, `deleter`. A more guided article about ACL can be found here:
 
-{% page-ref page="administration-concept.md" %}
+{% page-ref page="add-acl-rules.md" %}
```

Comment:
There is no `add-acl-rules.md` on the same directory. Guess you wanted `../../../guides/plugins/plugins/administration/add-acl-rules.md` instead? :-)